tasks/eth/021-holocene-protocol-versions/SignFromJson.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {ProtocolVersionsBump} from "script/verification/ProtocolVersionsBump.s.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Proxy} from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {ProtocolVersions, ProtocolVersion} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
```

Comment:
this can probably be cleaned up

Reply:
Significantly [4a57759](https://github.com/ethereum-optimism/superchain-ops/pull/428/commits/4a5775953b04a3f1b4c7f33d060dc035c0069811)